integration_tests/src/tests/mod.rs

```diff
@@ -35,6 +35,17 @@ pub use verified_claim_test::*;
 mod verifreg_remove_datacap_test;
 pub use verifreg_remove_datacap_test::*;
 mod withdraw_balance_test;
+use vm_api::VM;
 pub use withdraw_balance_test::*;
 mod move_partitions_test;
 pub use move_partitions_test::*;
+
+use lazy_static::lazy_static;
+use std::collections::BTreeMap;
+use std::sync::Mutex;
+
+type TestFn = fn(&dyn VM) -> ();
+
+lazy_static! {
+    pub static ref TEST_REGISTRY: Mutex<BTreeMap<String, TestFn>> = Mutex::new(BTreeMap::new());
```

Comment:
Uber-nits but....

1. Does this need to be a MAP? Honestly, you could probably get away with `Vec<(name, fn)`.
2. You can probably get away with `&'static str` instead of `String`.

Although neither of these actually matter for performance.

Reply:
One "feature" of using a Map here means that tests are always executed in alphabetical order, which can make it slightly more deterministic in external repos and easier perhaps to build tools around.

Reply:
Yeah, seems reasonable. Just make sure we don't have issues with duplicates (using the module names _should_ fix this, I think).